apps/ledger-live-mobile/src/screens/FirmwareUpdate/index.tsx

```diff
@@ -537,12 +538,13 @@ export const FirmwareUpdate = ({
       );
     }
 
-    const error =
-      updateActionState.error ??
-      restoreAppsState.error ??
-      installLanguageState.error ??
-      staxLoadImageState.error ??
-      staxFetchImageState.error;
+    const error = hasRetriableErrors
```

Comment:
same here for the name: do we want to display an error if the error is retriable ? or the opposite ?

Reply:
we want to display them when they're retriable, maybe retriable is not the best name